sources/packages/web/src/services/AestService.ts

```diff
@@ -0,0 +1,53 @@
+import ApiClient from "@/services/http/ApiClient";
+import {
+  StudentDetail,
+  SearchStudentResp,
+  GetApplicationDataDto,
+} from "@/types";
+/**
+ * Client Service layer for Ministry APIs
+ */
+export class AestService {
```

Comment:
Is this service supposed to hold all API calls related to AEST portal?

Reply:
Removed as discussed